derive/src/lib.rs

```diff
@@ -17,7 +17,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 extern crate alloc;
-extern crate proc_macro;
```

Comment:
Good bye old friend.

Reply:
Hm I think the CI needs its rust updating

Reply:
Will file an update

Reply:
Done it :point_down: (use latest nightly) - I didn't notice the `.circleci` directory.